pages/linux/mocp.md

```diff
@@ -0,0 +1,36 @@
+# mocp
+
+> Music on Console (MOC) audio player.
+> More information: <https://moc.daper.net/>.
+
+- Launch the MOC terminal UI:
+
+`mocp`
+
+- Launch the MOC terminal UI in a specific directory:
+
+`mocp {{path/to/directory}}`
+
+- Add a specific song to the play queue while MOC is in the background:
```

Comment:
Hi @tom-on-the-internet.
I don't understand if this command starts `moc` in the background. is the terminal UI only launched in foreground when is executed without arguments?

Reply:
@navarroaxel I think you tagged the wrong person here.

To your question: You're right. In order to use arguments like `--stop` or `--play`, there has to be a running `MOC`server either in the background or in another shell. If you run any of those if there's no running server you get an error.